addons/web/static/src/search/control_panel/control_panel.js

```diff
@@ -606,8 +617,8 @@ export class ControlPanel extends Component {
     _sortEmbeddedActionDrop({ element, previous }) {
         const order = this.state.embeddedInfos.embeddedActions
             .map((el) => el.id)
-            .filter((el) => el !== false);
-        const elementId = Number(element.dataset.id);
+            .filter((el) => el !== false || this.isMainEmbeddedActionOrderModifiable);
+        const elementId = Number(element.dataset.id) || false;
```

Comment:
Why do we need that change? 🤔 🙂

Reply:
ah because `element.dataset.id` could be equals to `false`?

Reply:
Yep, because Number(false) returns 0, but we want false here